Fix tests after #1839 and plone.app.event#411
[erral]
